static/app/components/globalModal/components.tsx

```diff
@@ -81,28 +81,21 @@ interface ClosableHeaderProps extends React.HTMLAttributes<HTMLHeadingElement> {
  * Creates a ModalHeader that includes props to enable the close button
  */
 const makeClosableHeader = (closeModal: () => void) => {
-  const ClosableHeader: React.FC<ClosableHeaderProps> = ({
-    closeButton,
-    children,
-    ...props
-  }) => (
+  const ClosableHeader = ({closeButton, children, ...props}: ClosableHeaderProps) => (
     <ModalHeader {...props}>
       {children}
       {closeButton ? <CloseButton onClick={closeModal} /> : null}
     </ModalHeader>
   );
 
-  ClosableHeader.displayName = 'Header';
```

Comment:
This was originally here to help with enzyme, but is no longer needed and affects how some consumers use the types.

Reply:
does `ClosableHeaderProps` contain `children`?

Reply:
It does I believe or this wouldn't pass `tsc`